File: ZooJava/src/main/java/com/example/zoojava/utils/Globals.java
package com.example.zoojava.utils;

import com.example.zoojava.models.Employee;

import java.io.File;
import java.nio.file.Path;
import java.nio.file.Paths;

public record Globals() {
    public static Employee globalEmployee;
    private static final Path CURRENT = Paths.get("");
    public static final String DIR_CSV = CURRENT.toAbsolutePath() + File.separator + "csv";
    public static final String ANIMALS_CSV = DIR_CSV + File.separator + "animales.csv";
    public static final String EMPLOYEE_CSV = DIR_CSV + File.separator + "empleados.csv";
}

File: ZooKotlin/src/main/kotlin/com/example/zookotlin/utils/Globals.kt
package com.example.zookotlin.utils

import com.example.zookotlin.models.Employee
import java.io.File
import java.nio.file.Paths

object Globals {
    lateinit var globalEmployee: Employee
    private val CURRENT = Paths.get("")
    val DIR_CSV = CURRENT.toAbsolutePath().toString() + File.separator + "csv"
    val ANIMALS_CSV = DIR_CSV + File.separator + "animales.csv"
    val EMPLOYEE_CSV = DIR_CSV + File.separator + "empleados.csv"
}